src/pages/settings/ConfirmationScreen.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {Image, View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import Text from '../../components/Text';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import styles from '../../styles/styles';
+import confettiPop from '../../../assets/images/confetti-pop.gif';
+import Button from '../../components/Button';
+import FixedFooter from '../../components/FixedFooter';
+import * as Link from '../../libs/actions/Link';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import Navigation from '../../libs/Navigation/Navigation';
+
+const propTypes = {
+    /* Onyx Props */
+
+    /** Holds information about the users account that is logging in */
+    account: PropTypes.shape({
+        /** Whether or not two factor authentication is enabled */
+        requiresTwoFactorAuth: PropTypes.bool,
+    }),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    account: {},
+};
+
+const ConfirmationScreen = props => (
+    <>
+        <View style={[styles.flex1, styles.p5, styles.justifyContentCenter]}>
+            <View style={[styles.alignItemsCenter, styles.mb10]}>
```

Comment:
After using `justifyContentCenter` it was too lower from the center so I have added a bottom margin, still it isn't perfectly centered so if we want we can increase the margin or add some padding too

Reply:
> still it isn't perfectly centered so if we want we can increase the margin or add some padding too
 
 Can share some screenshots, with & without `mb10`

Reply:
@thesahindia Can you try to add better styling than adding `mb10`? 

Also @chiragsalian What do you think about this?

Reply:
I think the `mb10` is fine. What i would recommend is centralizing this style since it can be pretty useful elsewhere or if someone wants to improve on it later can easily find it, maybe a className of `screenCenteredContainer`? Thoughts?

Reply:
Yeah we can do that, will make the change.